samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -157,34 +157,63 @@ struct metric {
     } end;
 };
 
+#define NUM_SAMPLES_PER_ENGINE 3
+#define PMU_ENGINE_INVALID -1
+
+struct engine_metrics {
+     metric metrics[NUM_SAMPLES_PER_ENGINE]; /* 0 - I915_SAMPLE_BUSY/__I915_PMU_OTHER, 1- I915_SAMPLE_WAIT, 2 - I915_SAMPLE_SEMA */
+     uint64_t num_samples;
```

Comment:
we need to be consistent. Either rename s/metrics/samples/ or rename s/num_samples/num_metrics/

Reply:
Ok